### PR TITLE
Improve layout for pr diff/space view

### LINKED ISSUE #197: Space view risizing

### PR DESCRIPTION
**Pull request by txsmith**
'Space' view now uses more space

Fixes #197

### CODE REVIEW

File: src/projects/projects-page.html [hunk 1]
```diff
@@ -47,7 +47,8 @@
       }
       iron-pages {
         flex: 1;
-        overflow: auto;
+        overflow-y: auto;
+        overflow-x: hidden;
```
Comment (TimvdLippe) [inline src/projects/projects-page.html]: Why was this added?

**Comment by eanker** [comment]
It works on my screen.
Maybe we can extract an issue to manually change the width of the right bar, in order to make it more readable.

File: src/projects/pull-request/space/pull-request-space.html [hunk 2]
```diff
@@ -100,7 +97,6 @@
         if (!this.isAttached) {
           return;
         }
-        this.$.container.style.width = Math.min(800, this.parentNode.getBoundingClientRect().width / 2 - 100) + 'px';
```
Comment (TimvdLippe) [inline src/projects/pull-request/space/pull-request-space.html:103]: :fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire:

**Comment by txsmith** [comment]
LGTM!

File: src/projects/projects-page.html [hunk 3]
```diff
@@ -47,7 +47,6 @@
       }
       iron-pages {
         flex: 1;
-        overflow: auto;
```
Comment (TimvdLippe) [inline src/projects/projects-page.html:50]: This accidently broke the create pull request page.
